templates/org/team/new.tmpl

```diff
@@ -89,10 +89,10 @@
 									</thead>
 									<tbody>
 										{{range $t, $unit := $.Units}}
-											{{if ge $unit.MaxPerms 2}}
+											{{if ge $unit.MaxPerm 2}}
 												<tr>
 													<td>
-														<div {{if $unit.Type.UnitGlobalDisabled}}class="field tooltip" data-content="{{$.i18n.Tr "repo.unit_disabled"}}">{{- else -}}class="field"{{end}}>
+														<div {{if $unit.Type.UnitGlobalDisabled}}class="field tooltip" data-content="{{$.i18n.Tr "repo.unit_disabled"}}"{{- else -}}class="field"{{end}}>
```

Comment:
```suggestion
														<div class="field{{if $unit.Type.UnitGlobalDisabled}} tooltip" data-content="{{$.i18n.Tr "repo.unit_disabled"}}{{end}}">
```

Reply:
I don't think it's more clear for developers. The slight copy here is better I think.

Reply:
I'm fine with both approaches. I simply don't like redundancy.

Reply:
I have a feeling that gotemplates do not like that kind of templating. The reason is that escaping within an attribute and outwith an attribute has to be different (and different for different types of attribute too). Thus I don't think this would work.

Reply:
Really? Then I understood gotemplates wrong. I thought they would only replace every instance of `{{..}}` (and potentially what's between two subsequent templating hints. I didn't know that you can escape in gotemplates.

Reply:
https://pkg.go.dev/html/template#hdr-Contexts